_posts/2020-01-24-inception-pipeline-cdk.md

```diff
@@ -0,0 +1,395 @@
+---
+layout: post
+title: Cross-Account Inception Pipeline with AWS CDK
+date: 2020-01-20
+tags: codepipeline cloud-development-kit infrastructure-as-code aws cdk cross-account
+author: JK Gunnink
+image: img/inception-pipelines/seed_germination.png
+---
+
+The [inception pipeline][inception-pipeline-post] which Mechanical Rock's very own Pete Yandell
```

Comment:
Might be worth a couple of sentences to resell the business value of this? No click-ops, auditability, etc.?

Reply:
Great idea!